projects/docs/docs/commands/plugin.md

```diff
@@ -0,0 +1,46 @@
+---
+slug: '/commands/plugin'
+title: Interact with plugins
```

Comment:
What do you think about `Sharing via plugins` instead of `Interact with plugins`?

Reply:
sounds good!